spec/requests/api/v0/highlights_controller_spec.rb

```diff
@@ -5,6 +5,63 @@
 
   before { allow(Rails.application.config).to receive(:consider_all_requests_local) { false } }
 
+  describe 'GET /highlights' do
+    let!(:highlight1) { create(:highlight, user_uuid: user_uuid) }
+    let!(:highlight2) { create(:highlight, user_uuid: user_uuid) }
+    let!(:highlight3) { create(:highlight) }
```

Comment:
btw, you don't necessarily need to use the bang version of `let!` -- I only used it in that other spec b/c if I didn't my `expect{blah}.to change{Highlight.count}.by(1)` got messed up because the instantiation happened in the `blah` part.

Reply:
this set of tests (mostly) depends on these 3 test records present in the db, before the tests run.   the let! makes sure they run before the tests.